### DISCUSSION TITLE
I cannot get my application go through

### DISCUSSION THREAD

**Discussion by KadirOrcunAltunel** [community/community#143693] [GitHub Education]
### Hi everyone,

The system doesn't let me upload files and only allows me to take pictures and they turn out to be blurry! Can someone please fix this! 
<img width="1470" alt="Screenshot 2024-11-05 at 5 05 25 PM" src="https://github.com/user-attachments/assets/1bcdf8a2-1231-4bee-bf83-bf4433459391">

**Comment by lucatosc** [comment]
Please resolve the issue with the image upload system and improve the camera quality to prevent blurry pictures.
